skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldExecutorService.kt

```diff
@@ -140,7 +145,8 @@ data class SkaffoldExecutorSettings(
     val skaffoldProfile: String? = null,
     val workingDirectory: File? = null,
     val skaffoldLabels: SkaffoldLabels? = null,
-    val tailLogsAfterDeploy: Boolean? = null
+    val tailLogsAfterDeploy: Boolean? = null,
+    val imageRepoOverride: String? = null
```

Comment:
can we use the same wording as the field in the UI: so something like `defaultImageRepo`?